qiskit/providers/models/pulsedefaults.py

```diff
@@ -56,17 +56,17 @@ class PulseDefaultsSchema(BaseSchema):
     """Schema for PulseDefaults."""
 
     # Required properties.
-    qubit_freq_est = List(Number(), required=True,
-                          validate=Length(min=1))
-    meas_freq_est = List(Number(), required=True,
-                         validate=Length(min=1))
-    buffer = Integer(required=True, validate=Range(min=0))
-    pulse_library = Nested(PulseLibraryItemSchema, required=True, many=True)
-    meas_kernel = Nested(MeasurementKernelSchema, required=True)
-    discriminator = Nested(DiscriminatorSchema, required=True)
+    qubit_freq_est = List(Number(), required=False,
+                          validate=Length(min=1), missing=None)
```

Comment:
As discussed in private, can we avoid having `missing=`, and instead either encourage the backends to provide sensible values for the attributes, or handle them somewhere in the pulse layers in terra? The rationale is that having a "default" blurs the separation between models (acting as "dummy containers") and logic a bit, and requires consumers to not only take into account the models, but the defaults - and if they reach that point, they are probably better off handling that in a different layer altogether, closer to the one that implements the functionality that uses those parameters or similar.